evrApp/Db/evrbase.db

```diff
@@ -447,3 +441,94 @@ record(waveform, "$(P)Label-I") {
   field(NELM, "128")
   info(autosaveFields_pass1, "VAL")
 }
+
+record(mbbo, "$(P)PLL-Bandwidth-Sel") {
+    field( DESC, "EVR Evt Clock Bandwidth")
+    field( DTYP, "Obj Prop uint16")
+    field( OUT,  "@OBJ=$(OBJ), PROP=PLL Bandwidth")
+    field( PINI, "YES")
+    field( VAL,  "$(PLL-Bandwidth-Sel\=2)")
+    field( ZRST, "HM")
+    field( ONST, "HL")
+    field( TWST, "MH")
+    field( THST, "MM")
+    field( FRST, "ML")
```

Comment:
Perhaps there could be more meaningful labels?

Reply:
Perhaps the following could be used:
ML - Wide
MM - Medium Wide
MH - Nominal
HL - Medium Narrow
HM - Narrow

This is a bit tough to describe as the actual values depend on the input frequency.

Reply:
Even these would be an improvement.  I was thinking that the leading 'H' was for High (or wide) bandwidth, which appears to be exactly backwards.

Reply:
Actually, they represent the values of 2 three-state inputs. They can be High, Middle (floating), or Low.